php/php.editor/src/org/netbeans/modules/php/editor/verification/AbstractClassInstantiationHintError.java

```diff
@@ -110,7 +115,19 @@ public void visit(ClassInstanceCreation node) {
                 }
             }
         }
+    }
 
+    private boolean isDollaredName(ClassName className) {
```

Comment:
Don't we have already this method somewhere?

Reply:
No, It seems that we don't have it. (I searched "isDollared".)
I'll move it to `CodeUtils`.
Thanks!

Reply:
Moved it.